src/utils/io.js

```diff
@@ -6,13 +6,31 @@
 import axios from "axios";
 
 const saveBlob = async (data, name, type) => {
-  const link = document.createElement('a');
-  link.style.display = 'none';
-  document.body.appendChild(link);
   const blob = new Blob([data], { type });
-  link.href = URL.createObjectURL(blob);
-  link.download = name;
-  link.click();
+  // No foolproof way to detect node. So try loading what you require.
+  // If it quacks like a duck...
+  let fs, os, path;
+  try{
+    fs = require('fs');
+    os = require('os');
+    path = require('path');
+  }catch(_){}
+
+  if(fs && os && path){
+    // is node
+    const buffer = Buffer.from(await blob.arrayBuffer());
```

Comment:
Do we need a `const { Buffer } = require('node:buffer');` anywhere to load the node `Buffer` class?  I'm not a node expert so I'm looking at some [docs examples](https://nodejs.org/api/buffer.html#static-method-bufferfromarraybuffer-byteoffset-length).

Reply:
No. `Buffer` is a built-in class in node. Like `Math` in DOM environment.